crates/forge/tests/integration/gas.rs

```diff
@@ -747,11 +945,20 @@ fn events_cost() {
     // 156 range_check_builtin ~= 7
     // 6 gas for 50 event values
     // ~13 gas for 50 event keys
-    assert_gas(&result, "events_cost", 7 + 6 + 13);
+    // 0 l1_gas + 0 l1_data_gas + (7 + 6 + ~13) * (100 / 0.0025) l2 gas
```

Comment:
Maybe if we are asserting exactly 13, let's remove `~` from both lines?

Reply:
here's the thing - afaik it was not exactly 13 :D that's why I have left the ~, as far as I remember it was actually 13.2 (but I didn't dig into why it was that, I assumed You have made a good job calculating this and I only verified new calculations, no conversions).

Reply:
It was like this for a while now. Maybe you can create an issue to investigate this aferwards?

Reply:
And link it in code here

Reply:
done https://github.com/foundry-rs/starknet-foundry/issues/3078

Reply:
Can we add a todo in the code?